Makefile

```diff
@@ -127,9 +126,7 @@ srpm: $(TARFILE) $(RPM_NAME).spec
 	  --define "_srcrpmdir `pwd`" \
 	  $(RPM_NAME).spec
 
-rpm: $(RPMFILE)
-
-$(RPMFILE): $(TARFILE) $(RPM_NAME).spec
+rpm: $(TARFILE) $(RPM_NAME).spec
```

Comment:
I would really like to keep the current version where the phony `rpm` target is just convenience, and the actual file is used for the dependency. Otherwise we would be back in a situation where make can't check if dependencies are current, and thus `make check` would *always* rebuild the rpm and thus the VM even if nothing changed (i.e. iterating on tests just sucks).